docs/vote-escrow/realms.mdx

```diff
@@ -218,15 +222,24 @@ veHNT position to multiple networks (IOT or MOBILE) or set different lockup time
 positions.
 
 Be warned, splitting a [Landrush](#landrush) position after the initial 10 day period will result in
-losing the multiplier for the tokins being split or transfered off the position.
+losing the multiplier for the tokens being split or transfered off the position.
+
+You must undelegate before spliting a stake. You can redelegate after the split is complete.
 
 ### Transfer a Stake
 
-Stakes can be transfered from one position into another within the same account. Positions can only
-be transfered to positions with a greater or equal duration. Only undelegated positions can be
-transfered. Staked tokens cannot be moved to other accounts. Landrush positions cannot be
-transfered. This functionality is useful for consolidating positions. If no other compatible
-positions exist in an account, the "Transfer" button will be disabled for a position.
+Staked tokens cannot be moved to other wallets, neither can the NFT representing a stake be transfered 
+to another wallet. Stakes can only be transfered from one position into another within the same account. 
+Positions can only be transfered to positions with a greater or equal duration. This functionality is 
+useful for consolidating positions. 
+Landrush positions cannot be transfered into after the landrush period. Any veHNT transfered out of a 
```

Comment:
Reword more clearly, e.g. "More tokens cannot be transferred into a land rush position..."